instrumentation/opentelemetry-instrumentation-flask/tests/test_programmatic.py

```diff
@@ -185,46 +185,127 @@ def test_exclude_lists(self):
         self.assertEqual(len(span_list), 1)
 
 
-class TestProgrammaticCustomSpanName(
+# class TestProgrammaticCustomSpanName(
```

Comment:
Why comment this out?

Reply:
Hi Leighton, I've commented those two tests out because the Hooks essentially serve the same functionality. I can delete the commented out tests if you would like.

Reply:
Yes, removing them would be preferred :)